chart/openfaas/README.md

```diff
@@ -4,6 +4,13 @@
 
 [OpenFaaS](https://github.com/openfaas/faas) (Functions as a Service) is a framework for building serverless functions with Docker and Kubernetes which has first class support for metrics. Any process can be packaged as a function enabling you to consume a range of web events without repetitive boiler-plate coding.
 
+## Important notes
+
+If you are upgrading from pre-`6.0.0`, please...
```

Comment:
This should tell people that have `securityContext: true` how to upgrade by keeping the same settings as they had before.

Reply:
The migration should include step like: 
- getting the current setting with `helm get values openfaas`
- creating a my-values.yaml with the the above values
- remove securityContext
- how to setup the new securityContexts to match the old settings
- how to upgrade using the custom values file

Reply:
Isn't "_tweak the `securityContexts` field (...) according to your requirements_" clear and flexible enough? Users that are updating may have different requirements now that they can in fact fully tweak the security context for their pods and containers, and I don't think we should be somehow suggesting that they should keep their existing settings for a successful update?

Reply:
@stefanprodan I don't think we should be telling users how to operate Helm, especially because said instructions can quickly get outdated, which makes things worse. I think that everyone that is using the Helm chart knows their way around Helm, and has certainly performed an update before. If not, they can certainly ask for help over Slack, don't you think?

Reply:
I think we should make them aware that without following the migration steps their OpenFaaS control plane will have **no security context** at all and show them how they can keep the current settings.

Reply:
@stefanprodan users that care about that most probably have their pod security policies in place, in which case they'll promptly notice their OpenFaaS control plane has "_*no security context* at all_". In any case, I've updated the `README.md` file in a way I believe is clear enough. Could you please check?